iconoir.com/components/Footer.tsx

```diff
@@ -1,9 +1,28 @@
 import { PeaceHand } from 'iconoir-react';
+import { url } from 'inspector';
+import { Text } from 'react-native-svg';
 import styled from 'styled-components';
 import { LICENSE_LINK } from '../lib/constants';
 import { Logo, LogoContainer, LogoIcon } from './Header';
 import { NavigationItemContainer } from './NavigationItem';
+import { Text13, Text17 } from './Typography';
 
+export interface FooterCategoryProps {
+  category: string;
+  links: [{ name: string, url: string }];
+}
+function FooterCategory({ category, links }: FooterCategoryProps) {
+  return (
+    <FooterCategoryContainer>
+      <FooterCategoryTitle>{category}</FooterCategoryTitle>
+      <FooterCategoryLinks>
+        {links.map((link) => (
+          <FooterCategoryLink href={link.url}>{link.name}</FooterCategoryLink>
```

Comment:
When rendering React arrays, you need a unique `key` as a property on each component. In this case, using `link.name` for that probably works fine.

Reply:
Noting your suggestion. Thank you!

Reply:
FYI this is moreso [a React requirement than a suggestion.](https://beta.reactjs.org/learn/rendering-lists#keeping-list-items-in-order-with-key)